### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by KatrinDav**
small changes in styles and structure of page;

### CODE REVIEW

File: src/components/TopHeader.vue [hunk 1]
```diff
@@ -146,7 +164,11 @@ export default {
...
   }
+}
 
-  
+@media (min-width: 1140px) {
+  .nav-menu__item {
+    margin: 0 35px;
+  }
 }
 </style>
```
Comment (antanina-reznikava) [inline src/components/TopHeader.vue]: Here the new line is needed at the end of the file.

File: src/pages/HomePage.vue [hunk 2]
```diff
@@ -1,55 +1,65 @@
...
-      <div class="main_image"></div>
+
     </section>
     <div class="more-info">
       <p>soon <span>in the cinema</span></p>
-      <p class="showItems" @click="changeVisible">see all</p>
+      <p
+        class="showItems"
+        @click="changeVisible"
+      >{{isVisible ? 'hide all' : 'see all'}}</p>
```
Comment (antanina-reznikava) [inline src/pages/HomePage.vue]: What do you think about moving this line to the `computed` property?

File: src/pages/HomePage.vue [hunk 3]
```diff
@@ -60,199 +70,180 @@ export default {
...
-   .movies > .movie-card:nth-child(3) {
-    display: inline;
+    .movies > .movie-card:nth-child(3) {
+      display: inline;
+    }
   }
 }
-  
-}
 </style>
```
Comment (antanina-reznikava) [inline src/pages/HomePage.vue]: Here also a new line is needed

**Review by antanina-reznikava — CHANGES_REQUESTED** [review]
Look's great, I added a few small suggestions :)

File: src/components/TopHeader.vue (hunk 1 above)
Comment (KatrinDav) [reply src/components/TopHeader.vue]: Yesterday I changed setting in VSC, so lines are added now.

**Review by antanina-reznikava — APPROVED** [review]

**Pull request merged**